### DISCUSSION TITLE
about _beforeTokenTransfer hook in openzeppelin

### DISCUSSION THREAD

**Discussion by d4r3-w0lf** [Cyfrin/foundry-full-course-cu#2570] [General]
Can you guys help me what's the function of _beforeTokenTransfer hook in openzeppelin erc20s and what the location of this hook?

**Comment by EngrPips** [comment]
Hello @Nandeesh-D, what section of the course uses this hook so that I can check my codebase for it?

**Comment by d4r3-w0lf** [comment]
It's not covered in this course

**Comment by EngrPips** [comment]
> Can you guys help me what's the function of _beforeTokenTransfer hook in openzeppelin erc20s and what the location of this hook?

Have you tried checking the openzeppelin erc20 contract codebase to see if you will find this hook? this https://github.com/OpenZeppelin/openzeppelin-contracts is their official GitHub page.
